### DISCUSSION TITLE
profile overview bug

### DISCUSSION THREAD

**Discussion by Gaurav-Kumar-Soni** [community/community#69940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

on reloading **profile overview page** , the content that are not **link** are appearing as link.

**Comment by PriyanshuPz** [comment]
@Gaurav-Kumar-Soni Github got an update that now in readme.md file 
the text which  starts with 
```
### 
```
will hightlight if you dont want that then you can use this

```md
**Hi**
<strong>Hi</strong>
__Hi__
```

output:
**Hi**
<strong>Hi</strong>
__Hi__

**Comment by Gaurav-Kumar-Soni** [comment]
thanks for updating me
